src/App.tsx

```diff
@@ -40,28 +49,31 @@ function App() {
 
   return (
     <Flex h="100vh" pos="relative" direction="column">
-      <Box
-        zIndex={-1}
-        pos="absolute"
-        w="full"
-        h="full"
-        opacity={0.15}
-        backgroundImage="url('/keycapPattern.png')"
-      />
+      <Box zIndex={-1} pos="absolute" w="full" h="full" opacity={0.15} />
       {viewState === ViewState.Overview && (
-        <Overview onOpenSettingsModal={onOpen} />
+        <Overview onOpenSettingsModal={onOpenSettings} />
       )}
       {viewState === ViewState.Addview && (
         <MacroProvider>
-          <Macroview isEditing={false} onOpenSettingsModal={onOpen} />
+          <Macroview
+            isEditing={false}
+            onOpenMacroSettingsModal={onOpenMacroSettings}
+          />
         </MacroProvider>
       )}
       {viewState === ViewState.Editview && (
         <MacroProvider>
-          <Macroview isEditing={true} onOpenSettingsModal={onOpen} />
+          <Macroview
+            isEditing={true}
+            onOpenMacroSettingsModal={onOpenMacroSettings}
+          />
         </MacroProvider>
       )}
-      <SettingsModal isOpen={isOpen} onClose={onClose} />
+      <SettingsModal isOpen={isOpenSettings} onClose={onCloseSettings} />
```

Comment:
This is first implementaion of the new modal for macro-specific advanced settings with a coming soon tooltip to get some excitement going.